.github/ISSUE_TEMPLATE/unreproducible_package.yml

```diff
@@ -0,0 +1,180 @@
+            </picture>
+          </a>
+        </p>
+
+        Welcome to Nixpkgs. Please replace the **`Unreproducible Package: PACKAGENAME`** template above with the correct package name (As seen in the [NixOS Package Search](https://search.nixos.org/packages)).
+
+        > [!NOTE]
+        > This form is for reporting unreproducible packages. For more information, see the [Reproducible Builds Status](https://reproducible.nixos.org/) page.
+        > To report a package that fails to build entirely, please use the "Build Failure" form instead.
+
+        ---
+  - type: "dropdown"
+    id: "version"
+    attributes:
+      label: "Nixpkgs Version"
+      description: |
+        What version of Nixpkgs are you using?
+
+        > [!IMPORTANT]
+        > If you are using an older or stable version, please update to the latest **unstable** version and check if the package is still unreproducible before continuing this report.
+      options:
+        - "Please select a version."
+        - "- Unstable (25.05)"
+        - "- Stable (24.11)"
+        - "- Oldstable (24.05)"
+      default: 0
+    validations:
+      required: true
+  - type: "textarea"
+    id: "introduction"
+    attributes:
+      label: "Introduction"
+      description: |
+        This is a generic introduction to build reproducibility.
+        Please replace **PACKAGENAME** below with the canonical package name of the package, as you have done for the title above.
+      value: |
+        Building **PACKAGENAME** multiple times does not yield bit-by-bit identical
+        results, complicating the detection of Continuous Integration (CI) breaches. For
+        more information on this issue, visit [reproducible-builds.org](https://reproducible-builds.org/).
+
+        Fixing bit-by-bit reproducibility also has additional advantages, such as
+        avoiding hard-to-reproduce bugs, making content-addressed storage more effective
+        and reducing rebuilds in such systems.
+      default: 0
+    validations:
+      required: true
+  - type: "textarea"
+    id: "how-to-reproduce"
+    attributes:
+      label: "Steps to Reproduce"
+      description: |
+        This is a step-by-step instruction set meant for maintainers to debug the package that is failing to reproduce. You should also follow it to gather the `diffoscope` logs that will be needed below.
+        Please replace **<package>** (including the greater than/lesser than signs) below with the canonical package name of the package, as you have done for the introduction and the title above.
+      value: |
+        ### 1. Build the package
+
+        This step will build the package. Specific arguments are passed to the command
+        to keep the build artifacts so we can compare them in case of differences.
+
+        Execute the following command:
+
+        ```
+        nix-build '<nixpkgs>' -A <package> && nix-build '<nixpkgs>' -A <package> --check --keep-failed
```

Comment:
I think this can be quite confusing as `<nixpkgs>` can be easily misread as `<package>`...

Reply:
That was copied verbatim from the Reproducible Builds template. Maybe we should change it to UPPERCASE?